### PR TITLE
Add subscription extended

### PR DESCRIPTION
**Pull request by antoniobg**
## Motivation / Description
We're adding a new event type `SUBSCRIPTION_EXTENDED`.

## Changes introduced
- Document new event
- Add sample event for expiration, which was missing before 
-

### CODE REVIEW

**Comment by antoniobg** [comment]
@jefago I've added some basic documentation for the new event. I think we should also mention something about the particularities of this event for the Play Store, but I'm not able to phrase it in a simple but clear way. 

We're also missing a new section in the [Common Webhook Flows](https://www.revenuecat.com/docs/event-flows) page, do we have a template to generate those graphics?

**Review by jefago — APPROVED** [review]
I added something about the reasons for the `SUBSCRIPTION_EXTENDED` webhook.

For the common webhook flows, there's this Figma file here: https://www.figma.com/file/EWPq22jDErIW4KIrSYH9K1/Common-Webhook-Flows?type=whiteboard&node-id=0%3A1&t=cQ0TtqyhZZDJ11lg-1

File: docs_source/🔌 Integrations & Events/webhooks/event-types-and-fields.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ RevenueCat sends webhooks in response to events that occur in your app. Here the
     "10-5": "✅",
     "10-6": "❌",
     "11-0": "`SUBSCRIPTION_EXTENDED`",
-    "11-1": "An existing subscription has been extended (the expiration date of the current subscription period has been pushed back to the future).",
+    "11-1": "An existing subscription has been extended (the expiration date of the current subscription period has been pushed back to the future).\n\nThis event is fired when a Apple App Store or Google Play Store subscription is extended through the store's API. On the Google Play Store, this event can also sometimes fire when Google defers charging for a renewal by less than 24 hours (for unknown reasons). In this case, you will receive a `SUBSCRIPTION_EXTENDED` webhook, followed by either a `RENEWAL` or `BILLING_ISSUE` webhook within the next 24 hours.",
```
Comment (antoniobg) [inline docs_source/🔌 Integrations & Events/webhooks/event-types-and-fields.md:99]: :chefkiss:

**Comment by antoniobg** [comment]
> I added something about the reasons for the `SUBSCRIPTION_EXTENDED` webhook.
> 
> For the common webhook flows, there's this Figma file here: https://www.figma.com/file/EWPq22jDErIW4KIrSYH9K1/Common-Webhook-Flows?type=whiteboard&node-id=0%3A1&t=cQ0TtqyhZZDJ11lg-1

I cannot edit that Figma file, but I have created a timeline for it [here](https://www.figma.com/file/OJ8IULqKeu01haNImMNobl/Subscription-Extended-Event?type=whiteboard&node-id=0%3A1&t=RZUIWnHfbV4duJDv-1)

image

I have no idea how to upload this file to readme though.

**Comment by jefago** [comment]
> > I added something about the reasons for the `SUBSCRIPTION_EXTENDED` webhook.
> > For the common webhook flows, there's this Figma file here: https://www.figma.com/file/EWPq22jDErIW4KIrSYH9K1/Common-Webhook-Flows?type=whiteboard&node-id=0%3A1&t=cQ0TtqyhZZDJ11lg-1
> 
> I cannot edit that Figma file, but I have created a timeline for it [here](https://www.figma.com/file/OJ8IULqKeu01haNImMNobl/Subscription-Extended-Event?type=whiteboard&node-id=0%3A1&t=RZUIWnHfbV4duJDv-1)
> 
> image
> 
> I have no idea how to upload this file to readme though.

I added the flow to the main Figma, thanks for creating it

**Comment by antoniobg** [comment]
Thanks, @jefago. I've just added the new flow and I've copied the bit you added to the event description in there as well.

**Pull request merged**